lib/metaclient/meta_client.go

```diff
@@ -683,6 +684,63 @@ func (c *Client) GetMeasurementInfoStore(dbName string, rpName string, mstName s
 	return mst, nil
 }
 
+func (c *Client) RetryHeartbeat(id string, cqs []string) error {
```

Comment:
Can we use `retryExec` to replace with `RetryHeartbeat `

Reply:
`retryExec` only send `ExecuteRequestMessage`.